src/Round.js

```diff
@@ -0,0 +1,76 @@
+class Round {
+  constructor(generatedPlayers, clues) {
+    this.players = generatedPlayers;
+    this.clues = clues;
+    this.currentPlayer = this.players[0]; //Change to be last rounds player to answer last question correctly
+    this.currentGuess;
+    this.currentAnswer;
+    this.currentClue ;
+    this.guessCount = 0;
+  }
+
+  setCurrentClue(id, pointValue, question) {
+    this.currentClue = this.clues.find(clue => {
+      if(clue.categoryId === id && clue.pointValue === pointValue && clue.question === question) {
+        return true;
+      } else {
+        return false;
+      }
+    });
+  }
+       
+
+
+  takeGuess(guess) {
+    if(guess === this.currentClue.answer) {
+      this.handleGuess(guess, true);
+      this.nextClueHandler(true)
+    } else {
+      this.handleGuess(guess, false);
+      this.nextClueHandler(false);
```

Comment:
This conditional can be cleaned up quite a bit. I'd make a variable to store that true/valse value instead:

```js
let answeredCorrectly = guess === this.currentClue.answer;
this.handleGuess(guess, answeredCorrectly);
this.nextClueHandler(answeredCorrectly);
```